src/js/lottoApp.js

```diff
@@ -0,0 +1,105 @@
+import {$} from './utils/selector.js';
+import {AmountForm} from './components/AmountForm.js';
+import {LottoNumbers} from './components/LottoNumbers.js';
+import {LottoWinnigNumber} from './components/LottoWinnigNumber.js';
+import {LOTTO_PRICE_PER_UNIT} from './consts/lottoConsts.js';
+import {pickLottoNumbers} from './utils/ticketing.js';
+import {LottoResultModal} from './components/LottoResultModal.js';
+
+export function LottoApp($el) {
+    /**
+     * @type {Object}
+     * @property {Number} amount - 구매금액
+     * @property {[[Number]]} lottoNumbers - 추첨번호[]
+     * @property {[Number]} winningNumber - 당첨번호
+     * @property {Boolean} showResultModal - 당첨결과 modal open 여부
+     */
+    let state = {
+        amount: null,
+        lottoNumbers: [[]],
+        winningNumber: [null, null, null, null, null, null],
```

Comment:
이렇게 바꿀 수도 있지 않을까요 ?? ㅎㅎ 로또 당첨 갯수가 변동 될 수도 있으니까용..(그럴리는 없겠지만유..)
```suggestion
        winningNumber: new Array(6).fill(null);
```